src/java.base/share/classes/sun/security/pkcs/PKCS9Attribute.java

```diff
@@ -378,13 +378,13 @@ private void init(ObjectIdentifier oid, Object value)
         this.oid = oid;
         index = indexOf(oid, PKCS9_OIDS, 1);
         Class<?> clazz = index == -1 ? BYTE_ARRAY_CLASS: VALUE_CLASSES[index];
-        if (!clazz.isInstance(value)) {
+        if (clazz == null || !clazz.isInstance(value)) {
```

Comment:
If my reading of the current impl is correct, if clazz is null, the attribute is not supported. The error message seems a bit misleading as it's not really due to the value itself, but the attribute is not supported. Is it just for avoiding NPE and changing it into IAE?